From 869abafe43a9cdaa3de6008d35f384c573fed905 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sat, 12 Nov 2022 15:29:30 -0600
Subject: [PATCH 1/1] Set DefaultTimeoutStopSec to 5s

This is more suitable for a desktop system where most services should stop quickly and we don't want the end user to wait for shutdown to complete.
---
 src/core/system.conf.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/system.conf.in b/src/core/system.conf.in
index 96fb64d2c1..c9e2d302b4 100644
--- a/src/core/system.conf.in
+++ b/src/core/system.conf.in
@@ -42,7 +42,7 @@
 #DefaultStandardOutput=journal
 #DefaultStandardError=inherit
 #DefaultTimeoutStartSec=90s
-#DefaultTimeoutStopSec=90s
+DefaultTimeoutStopSec=5s
 #DefaultTimeoutAbortSec=
 #DefaultRestartSec=100ms
 #DefaultStartLimitIntervalSec=10s
-- 
2.38.1

